src/showcase/app/core/icon-overview/icon-overview.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { CdnIcons, CdnIconService } from './cdn-icon.service';
+
+@Component({
+  selector: 'sbb-icon-overview',
+  templateUrl: './icon-overview.component.html',
+  styleUrls: ['./icon-overview.component.css'],
+})
+export class IconOverviewComponent {
```

Comment:
Is there a reason for the separation of CdnIconListComponent and this component?

Reply:
It's much more simple this way: less ng-ifs (check data availability), no share() in observable, less html :)

Reply:
Now you have increased complexity a bit with more files.
But it's fine this way.